contracts/ClaimsManager.sol

```diff
@@ -294,7 +283,15 @@ contract ClaimsManager is
         string calldata policy,
         string calldata metadata
     ) external override returns (bytes32 policyHash) {
-        require(claimant == msg.sender, "Sender not claimant");
+        require(
+            claimant == msg.sender ||
+                manager == msg.sender ||
+                IAccessControlRegistry(accessControlRegistry).hasRole(
+                    adminRole,
+                    msg.sender
+                ),
```

Comment:
Admins can also downgrade policies. An admin can already do many things to block payouts (revoke the arbitrator role, spoof the API3 price to be very high, etc.) so being able to do this shouldn't be an issue and can be used to revert mistakes.